client/layout/masterbar/bell-straight-icon.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+export const BellIcon = (): React.ReactNode => (
```

Comment:
Do we need all of `React` just to use `ReactNode` type?

Could we import just the type instead?

```suggestion
import type { ReactNode } from 'react'

export const BellIcon = (): ReactNode => (
```

Reply:
Replaced this with function instead : )